client/db_test.go

```diff
@@ -266,7 +266,7 @@ func ExampleTx_Commit() {
 	// 1/0: ab=2
 }
 
-func ExampleDB_Insecure() {
+func ExampleDB_second() {
```

Comment:
`_lowercase` is correct when not specifying a method that this example is for. Otherwise it's skipped by godoc (currently is). So `ExampleDB_insecure` sounds fine. I'm not sure why renaming it to `second` is better.
When in doubt, `godoc -http` and taking a look at the corresponding example page is the safest bet.